lib/rialto/etl/readers/sparql_statement_reader.rb

```diff
@@ -11,13 +15,22 @@ class SparqlStatementReader
         include Enumerable
 
         attr_reader :settings, :input_stream
+        attr_accessor :insert_count
 
         def initialize(input_stream, settings)
           @settings = settings
           @input_stream = input_stream
+          @insert_count = 0
+        end
+
+        # Get the logger from the settings, or default to an effectively null logger
+        def logger
+          settings['logger'] ||= Yell.new(STDERR, level: 'gt.fatal') # null logger
         end
```

Comment:
How about adding whatever setting (I assume) you used during development to `config/settings.yml`? Just to make things explicit. This may necessitate adding another line to the .gitignore file, since we probably don't want to check in log files.

Reply:
Anything you can point me to a best practice on logging? I just cribbed the code from elsewhere in rialto-etl.

Reply:
Actually, I've changed my mind on this one.

I think it's better for us to merge this work, then use #154 to add robust application logging.